packages/api/src/env.js

```diff
@@ -168,8 +175,61 @@ export function envAll (req, env, ctx) {
     )
   }
 
-  // via https://stripe.com/docs/api/payment_methods/object
-  // this can be used to mock realistic values of a stripe.com paymentMethod id
-  // after fulls tripe integration, this may not be needed on the env
-  env.mockStripePaymentMethodId = 'pm_1LZnQ1IfErzTm2rETa7IGoVm'
+  Object.assign(env, (env.ENV === 'test') ? createTestBillingEnv() : createStripeBillingContext(env))
+}
+
+/**
+ * @typedef {object} StripeDerrivedEnv
+ * @property {string} stripeSecretKey
+ */
+
+/**
+ * @param {Pick<Env, 'STRIPE_SECRET_KEY'|'db'>} env
+ * @returns {import('./utils/billing-types').BillingEnv & StripeDerrivedEnv}
+ */
+function createStripeBillingContext (env) {
+  const stripeSecretKey = env.STRIPE_SECRET_KEY
+  assert.ok(stripeSecretKey, 'env STRIPE_SECRET_KEY is required')
```

Comment:
sure

Reply:
83431fc2fd01ab1fd0cea84b077e146c9dbac0f6

Reply:
Assert will throw `AssertionError` and in the error handler we will need to define what user error we want to send (and whether it should have 500 code or 4xx code). We could get all this with assert, but I think it would make things more complex with current code